File: apps/java/io/dustin/apps/common/utils/Runnable.java
package io.dustin.apps.common.utils;

@FunctionalInterface
public interface Runnable {
    /*
    () -> void
    Runnable을 이용하여 트랜잭션을 분리할 수 있다고 한다.
     */
    public abstract void run();
}

File: JavaToKotlin/apps(kotlin)/src/main/kotlin/io/dustin/apps/common/utils/Runnable.kt
package io.dustin.apps.common.utils

import kotlin.jvm.JvmRecord
import kotlin.Throws

@FunctionalInterface
interface Runnable {
    /*
    () -> void
    Runnable을 이용하여 트랜잭션을 분리할 수 있다고 한다.
     */
    fun run()
}
